packages/cozy-sharing/src/components/autosuggest.styl

```diff
@@ -30,6 +30,9 @@
     padding .5rem 0
     list-style none
 
+.suggestionHighlighted
+    background var(--paleGrey)
```

Comment:
We don't want to use named color anymore. You should use their semantic name. (I don't know them)

Reply:
à vérifier dans les maquettes mais de tête souvent le paleGrey c'est un backgroundDefault... ou dans les actions tu as hover, focus et selected...